Tests/scripts/validate_files_structure.py

```diff
@@ -382,14 +382,21 @@ def changed_docker_image(file_path):
 
 
 def validate_version(file_path):
-    change_string = run_git_command("git diff HEAD {0}".format(file_path))
-    is_incorrect_version = re.search("\+([ ]+)?version: (!-1)", change_string)
-    is_incorrect_version_secondary = re.search("\+([ ]+)?\"version\": (!-1)", change_string)
-
-    if is_incorrect_version or is_incorrect_version_secondary:
+    file_extension = os.path.splitext(file_path)[1]
+    version_number = -1
+    if file_extension == '.yml':
+        yaml_dict = get_json(file_path)
+        version_number = yaml_dict.get('commonfields', {}).get('version')
+    elif file_extension == '.json':
+        with open("./" + file_path) as json_file:
+            json_dict = json.load(json_file)
+            if isinstance(json_dict, 'dict'):
+                if 'version' in json_dict:
```

Comment:
how come `version` will not exist ?

Reply:
Not every json file has version from what i've been looking at the repo